backend/btrixcloud/crawlconfigs.py

```diff
@@ -843,6 +847,9 @@ async def prepare_for_run_crawl(self, cid: UUID, org: Organization) -> CrawlConf
         except:
             await self.readd_configmap(crawlconfig, org)
 
+        if org.readOnly:
+            raise HTTPException(status_code=400, detail="org_set_to_read_only")
```

Comment:
```suggestion
            raise HTTPException(status_code=403, detail="org_set_to_read_only")
```
Seems more consistent to also return 403 here.